web/src/css/icon.css

```diff
@@ -1,5 +1,5 @@
 .icons {
-  color: #326de6;
+  color: var(--icon-color);
```

Comment:
most of the changes are swapping out hardcoded or sass variable values for css variables.